package.json

```diff
@@ -9,8 +9,8 @@
     "url": "https://github.com/angular/angular.js.git"
   },
   "engines": {
-    "node": ">=8.12.0",
-    "yarn": ">=1.10.1",
+    "node": ">=12.14.1",
+    "yarn": ">=1.21.1",
```

Comment:
Is there a specific reason why `yarn@1.19.0` or some other version isn't good enough? 

`1.10.1` is pretty old, but should we really drop support for ~`1.16`+ (for example that latest `yarn` available in macPorts in `1.19.1`)? 

Is there a specific feature in `1.21.1` that we now require?

Reply:
Yarn 1.21.1 contains a security fix, I'd rather people didn't use older versions: https://blog.npmjs.org/post/189618601100/binary-planting-with-the-npm-cli.

Reply:
OK, that makes sense.